ImageViewer/Source/ImageViewer/ImageViewer.swift

```diff
@@ -350,13 +350,13 @@ public final class ImageViewer: UIViewController, UIScrollViewDelegate, UIViewCo
             self.closeButton.alpha = 0.0
             self.view.transform = CGAffineTransformIdentity
             self.view.bounds = (self.applicationWindow?.bounds)!
-            self.imageView.frame = self.parentViewFrameInOurCoordinateSystem
+            self.imageView.frame = self.displacedView.frame
```

Comment:
I do not consider this safe. You are now assuming the parent controller for displaced view and the imageViewer parentController have the same origin. Since the original approach was safer and was working reliably, I don't see reason to remove it.